### ISSUE TITLE
Desktop: Share error message has low contrast in dark mode

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#13520]
### Operating system

Linux

### Joplin version

3.5.5

### Current behaviour

Currently, the share error message has [1.59:1](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=FFB86C) contrast in dark mode:

<img width="426" height="330" alt="screenshot: Contrast checker: 'Cannot add an item as a child of a read-only item'" src="https://github.com/user-attachments/assets/6e465610-fcc3-4282-9029-c1ea91473c6d" />

### Expected behaviour

The text contrast should be at least 4.5:1.

**Comment by Aayushdev18** [comment]
Hi, I’d like to work on this issue. Could you please assign it to me?

**Comment by bwat47** [comment]
this might have been fixed in https://github.com/laurent22/joplin/commit/d8920840f2c8c3802caf70fbca8ab20279fdbb1b, I don't have joplin cloud so I can't verify, but screenshot looks like Dracula theme and that banner looks like one of the colors that was tweaked (warningBackgroundColor: '#ffb86c' / colorWarn: '#ffb86c')

**Comment by laurent22** [comment]
Thanks for looking into it @bwat47!
